### PR TITLE
AST-3188 - Category Description Not Working in RTL mode

### PR DESCRIPTION
**Pull request by Sudhanshu-1508**
### Description
Category description for uncategorized not working as expected in RTL mode

### How has this been tested?

Add a description to the Uncategorized from the Category  and check in RTL mode

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/modules/posts-structures/css/archive-dynamic.css.php [hunk 1]
```diff
@@ -39,11 +39,7 @@ function astra_post_archive_structure_dynamic_css( $dynamic_css, $dynamic_css_fi
 	$layout_type     = astra_get_option( 'ast-dynamic-archive-' . $current_post_type . '-layout', 'layout-1' );
 	$layout_2_active = ( 'layout-2' === $layout_type ) ? true : false;
 
-	if ( $layout_2_active ) {
-		$selector = '.ast-archive-entry-banner[data-post-type="' . $current_post_type . '"]';
-	} else {
-		$selector = 'body.archive .ast-archive-description';
-	}
+	$selector = ( $layout_2_active ) ? '.ast-archive-entry-banner[data-post-type="' . $current_post_type . '"]' : '';
```
Comment (imnavanath) [inline inc/modules/posts-structures/css/archive-dynamic.css.php:42]: @Sudhanshu-1508 removing `'body.archive .ast-archive-description'` selector will break other stylings which currently working fine for layout-1. 
PS: That's the reason following test case is failing https://share.getcloudapp.com/Wnu6bv6w do check other way to fix it.

File: inc/modules/posts-structures/css/archive-dynamic.css.php [hunk 2]
```diff
@@ -43,6 +43,10 @@ function astra_post_archive_structure_dynamic_css( $dynamic_css, $dynamic_css_fi
 		$selector = '.ast-archive-entry-banner[data-post-type="' . $current_post_type . '"]';
 	} else {
 		$selector = 'body.archive .ast-archive-description';
+
+		if( is_rtl() ) {
+			$selector = ' .ast-separate-container .ast-author-box';
```
Comment (imnavanath) [inline inc/modules/posts-structures/css/archive-dynamic.css.php:48]: @Sudhanshu-1508 do you want to concatenate the selector or taking as new in RTL?

File: inc/modules/posts-structures/css/archive-dynamic.css.php [hunk 3]
```diff
@@ -42,7 +42,10 @@ function astra_post_archive_structure_dynamic_css( $dynamic_css, $dynamic_css_fi
 	if ( $layout_2_active ) {
 		$selector = '.ast-archive-entry-banner[data-post-type="' . $current_post_type . '"]';
 	} else {
-		$selector = 'body.archive .ast-archive-description';
+		$selector = 'body.archive .ast-archive-description ';
+		if( is_rtl() ) {
+			$selector .= '.ast-separate-container .ast-author-box';
```
Comment (imnavanath) [inline inc/modules/posts-structures/css/archive-dynamic.css.php:47]:
```suggestion
			$selector .= ', .ast-separate-container .ast-author-box';
```

Comment (imnavanath) [reply inc/modules/posts-structures/css/archive-dynamic.css.php:47]: @Sudhanshu-1508 is this problem is for layout-1 only?

**Comment by Sudhanshu-1508** [comment]
This is not an issue, there's a setting available for this in the customizer.

**Pull request closed without merging**